draft-ietf-dots-telemetry.xml

```diff
@@ -4355,7 +4383,7 @@ module ietf-dots-telemetry {
         "Can be a telemetry-setup or telemetry data.";
       case telemetry-setup {
         description
-          "Indicates the message is about telemetry.";
+          "Configuration information for telemetry messages.";
```

Comment:
```suggestion
          "Indicates the message is about telemetry steup.";
```

I prefer to avoid using "configuration" as we are manipulating yang data structure not configuration data nodes.